<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color : rgb(0, 0, 0)</string>
   </property>
   <widget class="QLabel" name="imgPanel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>763</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>925</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(255,255,255)
</string>
    </property>
    <property name="text">
     <string>잠시후</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>960</y>
      <width>1101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom : 3px solid rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="now_num">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>1000</y>
      <width>1101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>111  222  333  555-1  1242  12  513 1234</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bus_num">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>90</y>
      <width>371</width>
      <height>811</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <pointsize>80</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
line-height:1%</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style=&quot;line-height:1.15&quot;&gt;&lt;span&gt;312&lt;br&gt;53&lt;br&gt;1237&lt;br&gt;536-1&lt;br&gt;243&lt;br&gt;1234-1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>865</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(255,255,255)
</string>
    </property>
    <property name="text">
     <string>노선번호</string>
    </property>
   </widget>
   <widget class="QLabel" name="bus_time">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>90</y>
      <width>231</width>
      <height>811</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <pointsize>80</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
line-height:1%</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style=&quot;line-height:1.15&quot;&gt;&lt;span&gt;2분&lt;br&gt;3분&lt;br&gt;12분&lt;br&gt;30분&lt;br&gt;15분&lt;br&gt;1분&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1175</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(255,255,255)
</string>
    </property>
    <property name="text">
     <string>도착예정</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(255,255,255)
</string>
    </property>
    <property name="text">
     <string>현재위치</string>
    </property>
   </widget>
   <widget class="QLabel" name="bus_nolo">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>130</y>
      <width>541</width>
      <height>751</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
line-height:1%</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style=&quot;line-height:2.57&quot;&gt;&lt;span&gt;가나다라마바사(5전)
&lt;br&gt;가나다라마바사(5전)
&lt;br&gt;가나다라마바사(5전)
&lt;br&gt;가나다라마바사(5전)
&lt;br&gt;가나다라마바사(5전)
&lt;br&gt;가나다라마바사(5전)
&lt;br&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <zorder>bus_nolo</zorder>
   <zorder>imgPanel</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>now_num</zorder>
   <zorder>label_3</zorder>
   <zorder>bus_time</zorder>
   <zorder>bus_num</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
